lib/src/protos/git_store.proto

```diff
@@ -16,10 +16,22 @@ syntax = "proto3";
 
 package git_store;
 
+message TreeConflict {
+  repeated bytes removes = 1;
+  repeated bytes adds = 2;
+}
+
 message Commit {
   repeated bytes predecessors = 2;
   bytes change_id = 4;
 
+  oneof root_tree {
+    bytes resolved = 3;
+    TreeConflict conflict = 1;
+  }
+  // TODO(#1624): delete when we assume that all commits use this format
+  bool tree_conflict_format = 10;
```

Comment:
Perhaps `tree_level_conflicts` or `new_tree_conflict_format` might be a little clearer.

Reply:
I renamed it to `uses_tree_conflict_format` (`tree_level_conflicts` sounds like it might be a list of conflicts)